src/lean_spec/subspecs/containers/block/block.py

```diff
@@ -66,6 +66,18 @@ class BlockAndVote(Container):
     proposer_attestation: ValidatorAttestation
     """The proposer's vote corresponding to this block."""
 
+class BlockAndSignatures(Container):
+    """Bundle containing a block and its attestation signatures list."""
+
+    block: Block
+    """The proposed block message."""
+
+    signatures: BlockSignatures
```

Comment:
Maybe I got confused but does this mean we have signatures in `BlockAndSignatures.signatures` and also in `Block.body.attestations` which contains a list of `SignedVote`?